draft-ietf-quic-transport.md

```diff
@@ -2608,11 +2654,10 @@ discarded upon receipt.  This avoids potential ambiguity about which STREAM
 frames count toward flow control.
 
 Upon receipt of a STOP_SENDING frame on a stream in the "open" or "half-closed
-(remote)" states, an endpoint MUST send a RST_STREAM with an error code of
-QUIC_RECEIVED_RST.  If the STOP_SENDING frame is received on a stream that is
-already in the "half-closed (local)" or "closed" states, a RST_STREAM frame MAY
-still be sent in order to cancel retransmission of previously-sent STREAM
-frames.
+(remote)" states, an endpoint MUST send a RST_STREAM frame.  If the STOP_SENDING
```

Comment:
Actually, the point here is that the *application* sends the RST_STREAM.  The transport can't send RST_STREAM.  The transport can't do that without risking putting the application into a bad state.  I will change this to make it clearer.

Actually, I'm going to raise this on the list.  It seems like STOP_SENDING could be made an application-layer construct and we could avoid that complexity at the transport layer entirely.  I'm trying to see how visibility at the transport layer would be of value, but I'm drawing a blank.